partiql-ast/src/main/java/org/partiql/ast/Identifier.java

```diff
@@ -35,11 +41,131 @@ public <R, C> R accept(@NotNull AstVisitor<R, C> visitor, C ctx) {
     }
 
     @NotNull
-    public String getSymbol() {
-        return this.symbol;
+    public List<Simple> getQualifier() {
+        return this.qualifier;
+    }
+
+    @NotNull
+    public Simple getIdentifier() {
+        return this.identifier;
+    }
```

Comment:
Can you indicate that this is the right-most identifier in the Javadocs? AKA:
> For `a.b.c`, this method would return `c`.

Reply:
Yeah, I have another PR coming soon that updates the docs before the final release. I'll add this note.

Reply:
https://github.com/partiql/partiql-lang-kotlin/pull/1704 adds the comment/